src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java

```diff
@@ -12,6 +13,6 @@
 public class MixinMinecraftServer {
     @Inject(at = @At("RETURN"), method = "createCommandManager")
     public void intercept(CallbackInfoReturnable<CommandManager> infoIn) {
-
+        infoIn.getReturnValue().registerCommand(new CommandHolo());
```

Comment:
we should just get commands-v1 working properly :p

Reply:
commands v1 doesn't add it to /help

Reply:
make it do it

Reply:
concern
i don't think it's possible without breaking api

Reply:
no one uses v1 yet

Reply:
(properly)

Reply:
explain that to @shedaniel 
do you think anyone used those seperate registry events? no
did I still have to keep them? yes

Reply:
huh